### DISCUSSION TITLE
If ctb can be operated by arrow keys on the phone screen.

### DISCUSSION THREAD

**Discussion by CreeperJIMM** [ppy/osu#22549] [Ideas] [answered]
I play ctb on keyboard was left hand press dash, right hand press left and right keys.
But when I play ctb on my phone, I can't get used to it.

If the dash key can control by one hand, the other hand to move.
it will make me more used to it. (like the picture below)
<img  src="https://cdn.discordapp.com/attachments/733177408307331204/1072813772986650694/ctb.png">

[By the way]
Now the right button will accidentally press the pause button.
This affects the gaming experience.
<img src="https://cdn.discordapp.com/attachments/733177408307331204/1072809861424877599/Screenshot_20230208_172244.jpg">

**Comment by mk56-spn** [comment, marked as answer]
more customisation will be coming to touch controls so this will likely be possible in the future
